sdk/storage/azure-storage-file-datalake/tests/test_datalake_service_client.py

```diff
@@ -348,3 +350,148 @@ def test_connectionstring_without_secondary(self):
         assert client.url == 'https://foo.dfs.core.windows.net/fsname/dname'
         assert client.primary_hostname == 'foo.dfs.core.windows.net'
         assert not client.secondary_hostname
+
+    @DataLakePreparer()
+    def test_file_system_encryption_scope_from_service_client(self, datalake_storage_account_name, datalake_storage_account_key):
```

Comment:
These tests are all repeated in `test_file_system` right? If so, I think maybe we should leave out all these tests from the service client since the service client is just a pass through. Testing this manually (and not checking it in) is enough, I think. You could leave just this first one if you would like.

Sorry to make you undo some work here, but I think it'll be good to cut down the number of tests. Especially since we eventually have to migrate them to test proxy :P

Reply:
No worries! At least this also means I implicitly manually tested it 😉 just have to remove them from the test file now 😛